modules/utils/2.0/config/default.yaml

```diff
@@ -0,0 +1,27 @@
+lcr-modules:
+    utils:
+        inputs:
+            bed: "__UPDATE__"
+
+        options:
+            bam_sort: ""
+            bam_markdups: ""
```

Comment:
While you're updating the `utils` module, could you add `--overflow-list-size 600000` here? This should help address the issue raised in #58.